test/e2e-v2/cases/mongodb/otel-collector-config.yaml

```diff
@@ -0,0 +1,47 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+receivers:
+  prometheus:
+    config:
+     scrape_configs:
+       - job_name: 'mongodb-monitoring'
+         metrics_path: "/metrics"
+         scrape_interval: 10s
+         static_configs:
+           - targets: ['mongodb-exporter-1:9216','mongodb-exporter-2:9216']
```

Comment:
It looks like need to start an exporter for each MongoDB node, right?

Reply:
yes, one exporter for one mongodb node.

Reply:
I remember otel collector could have an extra label for fetching source?
@wankai123 @innerpeacez How do we do this for other otel based metrics?

Reply:
Could try `relabel_configs` in OTEL, if necessary

Reply:
> I remember otel collector could have an extra label for fetching source? @wankai123 @innerpeacez How do we do this for other otel based metrics?

Labels can only work for the upstream, and the upstream for MongoDB is the exporter. 

The exposure method of exporters may vary for different servers. For example, for Elasticsearch, only one exporter is needed, which will retrieve data through the ES cluster level RESTful API. As for RabbitMQ, it may not require a separate exporter since its nodes already have this capability (after a certain version).

Reply:
> Could try `relabel_configs` in OTEL, if necessary

match and replace `service_instance_id` to the MongoDB's name.

Reply:
`relabel` based on the premise that mongodb or mongodb-exporter has a label represent it's node name or ip addr, but it lacked currently.

Reply:
> `relabel` based on the premise that mongodb or mongodb-exporter has a label represent it's node name or ip addr, but it lacked currently.

I mean replace label `service_instance_id` value(mongodb-exporter-1:9216) to the fixed MongoDB's name (`mongodb-1` eg.)

Reply:
This is not a big deal to me, usually the exporter is deployed as a sidecar and thus the mongodb and the exporter share the same instance address

Reply:
I will give a relabel demo in otel-collector-config.yaml of e2e case. If someone want to change the `service_instnce_id` manually he can reference it.